src/libcrun/utils.c

```diff
@@ -1449,6 +1449,7 @@ run_process_with_stdin_timeout_envp (char *path, char **args, const char *cwd, i
 
   if (timeout > 0)
     {
+      sigemptyset (&mask);
```

Comment:
good catch.  Let's do this before we enter the `if (timeout > 0)` block

Reply:
seems that mask is not used when timeout == 0, but fine